components/calendar/src/any_calendar.rs

```diff
@@ -599,6 +677,25 @@ impl AnyCalendarKind {
         }
     }
 
+    /// Convert to a BCP-47 `Value`
+    pub fn as_bcp47_value(&self) -> Value {
+        match *self {
+            AnyCalendarKind::Gregorian => value!("gregory"),
+            AnyCalendarKind::Buddhist => value!("buddhist"),
+            AnyCalendarKind::Japanese => value!("japanese"),
+            AnyCalendarKind::JapaneseExtended => value!("japanext"),
+            AnyCalendarKind::Indian => value!("indian"),
+            AnyCalendarKind::Coptic => value!("coptic"),
+            AnyCalendarKind::Iso => value!("iso"),
+            AnyCalendarKind::Ethiopic => value!("ethiopic"),
+            AnyCalendarKind::Ethioaa => value!("ethioaa"),
+        }
+    }
+
+    /// Extract the calendar component from a [`Locale`]
+    ///
+    /// Will NOT perform any kind of fallbacking and will error for
```

Comment:
Nit (optional): just delete the method, rather than add this scary notice. It doesn't really do anything that can't be accomplished directly from Locale APIs.

Reply:
I tried removing it but it was used in a test. I thought about it more, and it's not actually that scary a notice; I think having this method is useful. Yes, it accomplishes things done with Locale APIs, but they're fiddly, you have to handle a bunch of error cases on the way. It's pretty rusty to have methods like that for convenience, and I think it's rather convenient to be able to check if a locale specified a known calendar.